src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -0,0 +1,474 @@
+          v-if="!!multisigAccount"
+          class="multisig-account-created"
+        >
+          <div class="message">
+            {{ $t('modals.creatingMultisigAccount.vaultCreatedMessage') }}
+          </div>
+          <AvatarWithChainName
+            :address="multisigAccount.gaAccountId"
+            show-address
+          />
+        </div>
+      </div>
+    </template>
+    <template v-else>
+      <div class="content">
+        <h2 class="title">
+          {{ $t('modals.createMultisigAccount.title') }}
+        </h2>
+        <div
+          v-for="(signer, index) in signers"
+          :key="`signer${index}`"
+        >
+          <InputField
+            v-model.trim="signer.address"
+            v-validate="{
+              required: true,
+              name_registered_address_or_url: true,
+            }"
+            :label="getSignerLabel(index)"
+            :placeholder="$t('modals.createMultisigAccount.signerInputPlaceholder')"
+          >
+            <template #label-after>
+              <a
+                class="scan-button"
+                @click.prevent="openScanQrModal(index)"
+              >
+                <QrScanIcon />
+              </a>
+            </template>
+          </InputField>
+        </div>
+
+        <div class="signers-add-wrapper">
+          <BtnText
+            :icon="PlusCircle"
+            :text="$t('modals.createMultisigAccount.addSigner')"
+            @click="addNewSigner"
+          />
+
+          <BtnHelp
+            :title="$t('modals.createMultisigAccount.addSignerHelpTitle')"
+            :msg="$t('modals.createMultisigAccount.addSignerHelpMsg')"
+          />
+        </div>
+
+        <div class="consensus">
+          <div class="description">
+            {{ $t('modals.createMultisigAccount.consensusRequiredDesc') }}
+          </div>
+
+          <div class="signers-count">
+            <select
+              v-model="requiredNumOfConfirmations"
+              class="num-of-singers-selector"
+            >
+              <template
+                v-for="i of signers.length"
+              >
+                <option
+                  v-if="i != 1"
+                  :key="`option-item-${i}`"
+                  :value="i"
+                >
+                  {{ i }}
+                </option>
+              </template>
+            </select>
+
+            <i18n
+              path="modals.createMultisigAccount.consensusRequiredContent"
+              tag="div"
+              class="text"
+            >
+              <span>
+                {{ minNumOfSigners }}
+              </span>
+            </i18n>
+
+            <BtnHelp
+              :title="$t('modals.createMultisigAccount.consensusRequiredHelpTitle')"
+              :msg="$t('modals.createMultisigAccount.consensusRequiredHelpMsg')"
+            />
+          </div>
+        </div>
+      </div>
+    </template>
+    <template #footer>
+      <BtnMain
+        v-if="isCreatingVault || !!multisigAccount"
```

Comment:
!!multisigAccount you convert that value to boolean few time, so please use for that computed property